code/game/turfs/simulated/floor.dm

```diff
@@ -232,13 +232,27 @@ var/global/list/turf/simulated/floor/phazontiles = list()
 			return 0
 
 
-/turf/simulated/floor/attack_paw(mob/user as mob)
+/turf/simulated/floor/attack_paw(mob/user)
+	if (is_light_floor())
+		if (user.a_intent == I_HURT)
+			remove_floor_tile()
+			set_light(0)
+			spark(src, 1)
+			visible_message("<span class='warning'>\The [user] removes the light tile from \the [src]!</span>")
+			return
 	return src.attack_hand(user)
 
-/turf/simulated/floor/attack_animal(mob/user as mob)
+/turf/simulated/floor/attack_animal(mob/user)
+	if (is_light_floor())
+		if (user.a_intent == I_HURT)
+			visible_message("<span class='warning'>\The [user] removes the light tile from \the [src]!</span>")
```

Comment:
Could mention claws or paws here. Something like "The [user] claws at the tile's edge and pries it away from the [src]."